samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -56,12 +56,14 @@ static struct broadcast_sink_stream {
 	size_t loss_cnt;
 	size_t error_cnt;
 	size_t valid_cnt;
+	uint16_t in_buf_len;
+	uint8_t *in_buf;
```

Comment:
Since this is stored as a pointer, does this mean that streams will share a single buffer? 

Would it require too much memory to have a buffer per stream?